modules/react/src/components/export-video/export-video-panel-container.js

```diff
@@ -297,7 +301,8 @@ export class ExportVideoPanelContainer extends Component {
     adapter.animationManager.setKeyframes('kepler', {
       ...this.getFilterKeyframes(),
       ...this.getTripKeyframes(),
-      cameraKeyframe: this.getCameraKeyframes()
+      cameraKeyframe: this.getCameraKeyframes(),
+      getTimeRangeFilterKeyframes
```

Comment:
Does `getTimeRangeFilterKeyframes` need to update over time? If not, it isn't necessary to set it. Instead we can make it a member of KeplerFilterKeyframes provided at construction.

Reply:
Debugged a bit, looks like getTimeRangeFilterKeyframes a member of KeplerFilterKeyframes is enough.